coreos-base/oem-gce/files/files/pam-sshd-config

```diff
@@ -0,0 +1,3 @@
+account    requisite    pam_oslogin_login.so
+account    optional     pam_oslogin_admin.so
+session    optional     pam_mkhomedir.so
```

Comment:
It looks like this file will override `/lib/pam.d/sshd`, which configures SSH to use the `system-login` settings.  That seems like something we don't want to throw out completely, so this will probably need to have the additional rules mixed into the existing configuration.